### DISCUSSION TITLE
Hey Google returns web results not NightScout metrics

### DISCUSSION THREAD

**Discussion by mdoliver** [nielsmaerten/gluco-check#176] [Q&A] [answered]
Hi all,

I'm super keen to get this going. My issue is that after successful linking of my NightScout site via glucocheck.app, asking Google Assistant to talk to gluco check (or any glucocheck command for that matter), only returns web results or "here's what I found on the web".

What am I missing?

Many thanks.

**Comment by nielsmaerten** [comment, marked as answer]
Hi @mdoliver 

I seem to remember us sending a few emails back and forth about a year ago. Weren't you having a similar issue with Nightscout Status? You're in New Zealand, right?

This is a direct link to Gluco Check in Google Assistant's App Directory: https://assistant.google.com/services/a/uid/0000008351d3e7c0?source=web
Try tapping this link on your phone. It should show Gluco Check's page in the App Directory (like [this](https://i.imgur.com/FLHkKKi.png)).
Then try tapping one of the example questions. What happens if you do this?

**Reply by mdoliver** [reply]
Hi @nielsmaerten , yes that's me, good memory! I'm in NZ and I've just tried your links. When I tap on the Google question in your link it worked. Now when I go back and try the voice assistant, it all works fine. 

Maybe the link you sent me has livened something up for Google Assistant? Or maybe I just have to wait a certain amount of time (hours?) for changes to take effect.

FYI to get Google routines to work, I not only had to set my Assistant language settings to US English I also had to set my default Android OS language setting to US English and remove all other languages. 

Thanks so much for replying.

**Reply by nielsmaerten** [reply]
Thanks for the info on setting the language to US English. Looks like that indeed fixes the bug, or at least circumvents it.

For some reason some Assistant apps just don't work when the phone/assistant language is set to NZ English.
I double-checked and New Zealand is definitely on the list of countries where the app should be released, so I don't really know what's causing this.

I'm glad it's working for you now though!
